test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -35,6 +35,24 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
     }
+    
+    @Test
+    public void isAnyNull() throws Exception {
+        // no items
+        assertFalse(Utils.isAnyNull());
+        
+        // null item
```

Comment:
The comment `null item` is slightly ambiguous, since it could mean a single item with `null`, or the `items` varargs array is `null` (what is being tested).

Actually, putting aside the comment, we probably *do* need to test both cases.

So, something like this? (Tweak it as you see fit)

```java
// items varargs array is null
assertFalse(Utils.isAnyNull(null));

// one item, which is null
assertFalse(Utils.isAnyNull((Object) null));
```

Reply:
That's a good distinction I didn't consider. Added. Thank you.